src/heaps/mutable_binary_heap.jl

```diff
@@ -212,6 +212,18 @@ top(h::MutableBinaryHeap) = h.nodes[1].value
 
 pop!{T}(h::MutableBinaryHeap{T}) = _binary_heap_pop!(h.comparer, h.nodes, h.node_map)
 
+"""
+```
+update!{T}(h::MutableBinaryHeap{T}, i::Int, v::T)
+```
+
+### Args:
+* `h`: Heap derived from `AbstractMutableHeap`.
+* `i`: The handle to the element which needs to be updated.
+* `v`: The updated value.
+
+Heaps of type `AbstractMutableHeap` allows to change its values after being pushed through this function `update!`. 
```

Comment:
Hmm, I'm not so sure about this description but I don't have any experience with it, so re-writing it is proving difficult. But heres an attempt derived from the old documentation:

"Updates the value of an element in heap `h` (referred to by the handle i) with the value `v`"